<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormActFor</class>
 <widget class="QWidget" name="FormActFor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>407</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>407</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>815</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">循环</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelParaTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#labelParaTitle
{
	color: black;
	border-image: url(:/img/widget/Panel.png) 0 4 0 4;
	border-left: 4px transparent;
	border-right: 4px transparent;
}
</string>
     </property>
     <property name="text">
      <string>循环指令参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameActFor">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="IsMainArea" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="xRadioButton" name="radioConst">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>循环次数为常数</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="xSpinBox" name="BoxConst">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string notr="true"/>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string notr="true"/>
        </property>
        <property name="prefix">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="HasBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>5</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="xLineEdit" name="lineEditVar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="xRadioButton" name="radioVar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>循环次数为变量</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacerDown">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>682</width>
          <height>626</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xRadioButton</class>
   <extends>QRadioButton</extends>
   <header>xradiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>xSpinBox</class>
   <extends>QSpinBox</extends>
   <header>xspinbox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioConst</tabstop>
  <tabstop>BoxConst</tabstop>
  <tabstop>radioVar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioConst</sender>
   <signal>toggled(bool)</signal>
   <receiver>BoxConst</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditVar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BoxConst</sender>
   <signal>clicked()</signal>
   <receiver>FormActFor</receiver>
   <slot>inputConst()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditVar</sender>
   <signal>clicked()</signal>
   <receiver>FormActFor</receiver>
   <slot>selectVar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>inputConst()</slot>
  <slot>selectVar()</slot>
 </slots>
</ui>
